TTS/trainer.py

```diff
@@ -640,10 +651,24 @@ def _optimize(
                 update_lr_scheduler = scale_prev <= scaler.get_scale()
                 loss_dict["amp_scaler"] = scaler.get_scale()  # for logging
         else:
+            if self.model.capacitron_vae:
```

Comment:
This is the sort of wonky but very essential part of the optimization process for Capacitron. This routine worked in the old implementation and I just ported it in here. It's essential to have both optimizers in the scope here.